index.d.ts

```diff
@@ -0,0 +1,711 @@
+declare interface AggregateError extends Error {
+  errors: any[];
+}
```

Comment:
do we still need to define it? It's a global object now:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/AggregateError

Reply:
we're still [using the library](https://github.com/semantic-release/semantic-release/blob/915f36cda7ed9bd2487d6dd5d7b32ca9b32d3dd3/package.json#L34) rather than the built-in. i think there is something that would make the switch a breaking change, but i do think we should migrate